src/resources/i18n/translations/en.json

```
@@ -605,11 +605,11 @@
   "Show your support": "Show your support",
   "Show Completed": "Show Completed",
   "Show lightning icons next to value for value enabled podcasts": "Show lightning icons next to value for value enabled podcasts",
-  "Sign Up": "Sign up",
-  "Sign Up Error": "Signup error",
+  "Sign Up": "Yes please",
+  "Sign Up Error": "Could not register account",
   "Sign up for a premium membership to use this feature": "Buy Podverse Premium to use this feature.",
   "Skip to next item in your queue": "Skip to next in queue",
-  "Skip track buttons are time jumps on lockscreen and bluetooth devices": "Clicking skip-track buttons on Bluetooth devices and the lockscreen makes jumps in playback.",
+  "Skip track buttons are time jumps on lockscreen and bluetooth devices": "Make jumps in playback with (software and hardware) skip-track buttons.",
```

```suggestion
  "Skip track buttons are time jumps on lockscreen and bluetooth devices": "Playback jumping with skip-track buttons on Bluetooth devices and the lockscreen",
```
A variant of the above. I am guessing buttons on cables also work (hence "hardware").